package.json

```
@@ -10,7 +10,8 @@
     "test": "jest --watchAll",
     "test:once": "jest",
     "build": "babel -d build --ignore __mocks__ src && node ./config/postbuild.js",
-    "serve": "node --harmony node_modules/sequelize-cli/lib/sequelize db:migrate --config build/config/database.js && node --harmony build/index.js"
+    "serve": "node --harmony node_modules/sequelize-cli/lib/sequelize db:migrate --config build/config/database.js && node --harmony build/index.js",
+    "blizztos": "npx babel-node scripts/BlizzardTosDelete.js"
```

```suggestion
    "blizztos": "babel-node scripts/BlizzardTosDelete.js"
```

I reckon this should work without needing the global babel-node